ui/app/templates/components/auth-form.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="auth-form" data-test-auth-form>
-  {{#if (or (this.error) (and this.waitingForOktaNumberChallenge (not this.cancelAuthForOktaNumberChallenge)))}}
+  {{#if (and this.waitingForOktaNumberChallenge (not this.cancelAuthForOktaNumberChallenge))}}
```

Comment:
I noticed while testing error handling in `AuthJwt` that this was catching all errors and also throwing an error of it's own since it was trying to lookup `this.error` as a helper. It doesn't look like the error prop is relevant for the condition so I removed it.